src/js/view/controls/controls.js

```diff
@@ -305,6 +309,7 @@ export default class Controls extends Events {
                     if (model.get('fullscreen')) {
                         api.setFullscreen(false);
                         this.playerContainer.blur();
+                        shouldFocusOnInactive = false;
```

Comment:
Does `shouldFocusOnInactive` need to be set to false here if it gets set by the blur eventListener? I honestly don't know.

Reply:
yeah it seems like you might want to do this before `blue` or `setFullscreen` is called.